src/python/CRABClient/CrabRestInterface.py

```diff
@@ -174,14 +174,27 @@ def makeRequest(self, uri=None, data=None, verb='GET'):
             url = url + '?' + data
 
         command = ''
-        command += 'curl -v -X {0}'.format(verb)
-        command += ' -H "User-Agent: %s/%s"' % (self['userAgent'], self['version'])
-        command += ' -H "Accept: */*"'
-        command += ' --data @%s' % path
-        command += ' --cert "%s"' % self['cert']
-        command += ' --key "%s"' % self['key']
-        command += ' --capath "%s"' % caCertPath
-        command += ' "%s" | tee /dev/stderr ' % url
+
+        # CRAB_useGoCurl env. variable is used to define how request should be executed
```

Comment:
add one line comment like this:
# same variable is used also inside CRABServer, we should keep name changes (if any) synchronized

Reply:
thanks, done